concepticondata/conceptlists/Hantgan-2022-351.tsv

```diff
@@ -0,0 +1,352 @@
+Hantgan-2022-351-295	295	SUGAR	sucre	862	SUGAR
+Hantgan-2022-351-296	296	SUN	soleil	1343	SUN
+Hantgan-2022-351-297	297	SWALLOW	avale	897	SWALLOW
+Hantgan-2022-351-298	298	SWEEP	balayer	230	SWEEP
+Hantgan-2022-351-299	299	SWEET	(goût) doux, sucré	717	SWEET
+Hantgan-2022-351-300	300	SWIM	nager	1439	SWIM
+Hantgan-2022-351-301	301	TAIL	queue	1220	TAIL
+Hantgan-2022-351-302	302	TALK	dire, parler	2708	TALK
+Hantgan-2022-351-303	303	TASTE	goûter	1373	TEA
+Hantgan-2022-351-304	304	TEN	thé	1515	TEN
+Hantgan-2022-351-305	305	TESTICLES	oeuf, testicule	797	TESTICLES
+Hantgan-2022-351-306	306	THICK	(bâton etc.) épais	1244	THICK
+Hantgan-2022-351-307	307	THIGH	cuisse	800	THIGH
+Hantgan-2022-351-308	308	THING	chose	738	THING
+Hantgan-2022-351-309	309	THINK	croire; réfléchir [cf. hò.gù, só:sê, lásá:bû, síkkê] -ex: à.bò .mí:ló [.ò.gú wìy-à/ he thinks they are doing the killing	2271	THINK
+Hantgan-2022-351-310	310	THORN	épine	124	THORN
+Hantgan-2022-351-311	311	THOUSAND	mille	1843	THOUSAND
+Hantgan-2022-351-312	312	THREE	trois	492	THREE
+Hantgan-2022-351-313	313	THROAT	cou, gorge, col	1346	THROAT
+Hantgan-2022-351-314	314	THROW	jeter	1456	THROW
+Hantgan-2022-351-315	315	TIE	attacher	1917	TIE
+Hantgan-2022-351-316	316	TOBACCO	tabac [related verb súndú 1a; SYN:táb-ò ; cf. hámpérê ]	974	TOBACCO
+Hantgan-2022-351-317	317	TOE	doigt, orteil	1389	TOE
+Hantgan-2022-351-318	318	TOMORROW	demain	1329	TOMORROW
+Hantgan-2022-351-319	319	TONGS	pinces (pour enlever les échardes); pinces (du forgeron)	272	TONGS
+Hantgan-2022-351-320	320	TONGUE	lâcher	1205	TONGUE
+Hantgan-2022-351-321	321	TOOTH	dent	1380	TOOTH
+Hantgan-2022-351-322	322	TOUCH	toucher	1892	TOUCH
+Hantgan-2022-351-323	323	TRAP (PITFALL)	piège	1075	TRAP (PITFALL)
+Hantgan-2022-351-324	324	TRAVERSE	casser, détruire ; traverser	666	RIVER
+Hantgan-2022-351-325	325	TREE	arbre	906	TREE
+Hantgan-2022-351-326	326	TROUGH	abreuvoir	205	TROUGH
+Hantgan-2022-351-327	327	TWENTY	vingt	1710	TWENTY
+Hantgan-2022-351-328	328	TWICE	deux fois	1546	TWICE
+Hantgan-2022-351-329	329	TWO	deux	1498	TWO
+Hantgan-2022-351-330	330	UNTIE	délier, lâcher (animal etc.)	352	UNTIE
+Hantgan-2022-351-331	331	URINE	urine	1142	URINE
+Hantgan-2022-351-332	332	VAGINA	vagin	1836	VAGINA
+Hantgan-2022-351-333	333	VILLAGE	côté, village, pays natal	930	VILLAGE
+Hantgan-2022-351-334	334	WAGES	payer, rembourser (qn)	1122	WAGES
+Hantgan-2022-351-335	335	WALL (OF HOUSE)	mur	933	WALL (OF HOUSE)
+Hantgan-2022-351-336	336	WATER	eau	948	WATER
+Hantgan-2022-351-337	337	WEAVE	tisser	133	WEAVE
+Hantgan-2022-351-338	338	WEEK	semaine	1469	WEEK
+Hantgan-2022-351-339	339	WET	complètement trempé, très mouillé	1726	WET
+Hantgan-2022-351-340	340	WHITE	blanc (de l'oeil, de l'oeuf)	1335	WHITE
+Hantgan-2022-351-341	341	WIFE	épouse	1199	WIFE
+Hantgan-2022-351-342	342	WIND	vent	960	WIND
+Hantgan-2022-351-343	343	WING	aile	1257	WING
+Hantgan-2022-351-344	344	WOMAN	femme	962	WOMAN
+Hantgan-2022-351-345	345	WOOD	bois	1803	WOOD
+Hantgan-2022-351-346	346	WORK	travailler	1803	WOOD
```

Comment:
Also a double mapping from the previous. This should probably be mapped to WORK.

Reply:
Yes.